src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/MediaContext.cs

```diff
@@ -1849,7 +1846,7 @@ private object InputMarkerMessageHandler(object arg)
                         // (TimeManager gets its tick time from MediaContext's IClock implementation).
                         // In the case where we can't query QPC or aren't doing interlocked presents,
                         // this will be equal to the current time, which is a good enough approximation.
-                        Rendering(this.Dispatcher, new RenderingEventArgs(_timeManager.LastTickTime));
+                        Rendering?.Invoke(this.Dispatcher, new RenderingEventArgs(_timeManager.LastTickTime));
```

Comment:
May I know when Render is null?

Reply:
@brunom There is already a null check line#1842

Reply:
@vishalmsft There is but the variable may have gone back to null in the meantime.